flow/core/params.py

```diff
@@ -1308,7 +1310,7 @@ def _multi_lane_headways_util(self, veh_id, edge_dict, num_edges, env):
         this_pos = self.get_position(veh_id)
         this_edge = self.get_edge(veh_id)
         this_lane = self.get_lane(veh_id)
-        num_lanes = env.scenario.num_lanes(this_edge)
+        num_lanes = env.k.scenario.num_lanes(this_edge)
 
         # set default values for all output values
         headway = [1000] * num_lanes
```

Comment:
This default value should be moved into the method definition as an optional param; otherwise it's unexpected behavior